sidebars.js

```diff
@@ -51,10 +51,22 @@ const sidebars = {
             id: "developer-docs/build/agents/index",
           },
           items: [
-            'developer-docs/build/agents/agent-dfinity',
-            'developer-docs/build/agents/ic-agent-dfinity'
+            {
+              type: "category",
+              label: "JavaScript",
```

Comment:
```suggestion
              label: "JavaScript Agent by DFINITY",
```
More agents will appear in this list. With just 'JavaScript' I think this will lead to an undiscovered page, or some naming clashes